### DISCUSSION TITLE
The laser leaves marks (engraved lines) when traveling back and forth

### DISCUSSION THREAD

**Discussion by MaiLes839** [arkypita/LaserGRBL#1139] [Q&A]
Hi, when I engrave an image like the below, the laser engrave unwanted lines when - how can I avoid this?
Thanks

![image](https://user-images.githubusercontent.com/72291216/95645795-42c86400-0acb-11eb-82d1-6ee5a4fcebb2.png)

**Comment by StuartB4** [comment]
Start LaserGRBL and click Connect.
Check grbl configuration on the Grbl menu. Look for $32 in the list and make sure it's set to 1.
Set $30 to 255.
Then click Write and Close. 

Check in the Settings on the Grbl menu.
Make sure Support PWM is enabled. It's the second tab from the left at the top.

After you've prepared your image with the brightness, contrast, engraving method, quality etc. 
Click Next and make sure Laser ON is set to M4.
S-MIN 0 and S-MAX 255.

**Comment by MaiLes839** [comment]
thanks, but unfortunately it didn't work, still engraving these lines...

**Comment by BeatSlayer** [comment]
Do they appear on the preview?
It doesn't really look like engraving while traveling because there are also some lines missing, could it be a connection issue?
Are you connecting at the recommended bitrate for your board?
Are you using a thick shielded usb cable away from the motor wires?

**Comment by MaiLes839** [comment]
Hi, here is the preview and they do not appear there (unless there is something I am missing):
![image](https://user-images.githubusercontent.com/72291216/95846966-ccd72d80-0d54-11eb-9930-20a93e5832d8.png)

I'm not sure I understand your question: "Are you connecting at the recommended bitrate for your board?" - where can I configure this?

Thanks!

**Comment by gmmanonymus111** [comment]
I think @BeatSlayer meant the baud rate (aka bitrate) of your connection. It can be set before connecting to your machine
![image](https://user-images.githubusercontent.com/9658243/95848069-d90bbc80-0d4d-11eb-9d63-db5ce4d05cb7.png)

But, as your machine connects (and LaserGRBL receives status messages), I would say the bitrate is fine.
It seems your engraver does not receive some of the commands sent by LaserGRBL. I would try with a different cable (most wired printers use the same kind, if you have one, you could try with that, they usually better in quality than chinese ones bundled with arduinos/engravers).
You may also try to lower the status report requests in LaserGRBL (`Grbl`->`Settings`), to either `Fast `or `Slow` from `UltraFast`.
![image](https://user-images.githubusercontent.com/9658243/95848797-d1004c80-0d4e-11eb-8f8b-173092dae88c.png)

**Comment by MaiLes839** [comment]
Thanks Alexs!
I changed to "Slow" but still didn't work, I will try changing the cable (it is actually not from China - they didn't send any cable with the machine, but I will try anyways...)
